packages/next/src/index.ts

```diff
@@ -34,6 +34,19 @@ export default class LogtoClient {
       }
     });
 
+  handleSignOut = (redirectUri = this.config.baseUrl): NextApiHandler =>
+    this.withIronSession(async (request, response) => {
+      const nodeClient = this.createNodeClient(request);
```

Comment:
Do we have to create a new `nodeClient` every time we handle sign-in or sign-out? Is it possible to maintain a singleton instance?

Reply:
No, they need to get `request` first, actually, each request is independent, they won't share anything (in memory) between API requests.